### PR TITLE
Check for verified attendance days

### PR DESCRIPTION
**Pull request by agundy**

### CODE REVIEW

File: attendance.js [hunk 1]
```diff
@@ -24,11 +24,11 @@ module.exports.getUserAttendance = function(userid, currentClassYear){
     }
 
     function largeGroupFilter(a) {
-        return a.user.$oid === userid && a.classYear.$oid === currentClassYear._id.$oid && !a.smallgroup && !a.bonusDay;
+        return a.user.$oid === userid && a.classYear.$oid === currentClassYear._id.$oid && !a.smallgroup && !a.bonusDay && a.verified;
     }
 
     function bonusDayFilter(a) {
-        return a.user.$oid === userid && a.classYear.$oid === currentClassYear._id.$oid && a.bonusDay;
+        return a.user.$oid === userid && a.classYear.$oid === currentClassYear._id.$oid && a.bonusDay && a.verified;
```
Comment (seveibar) [inline attendance.js:31]: wow, good thinking!

File: attendance.js [hunk 2]
```diff
@@ -24,11 +24,11 @@ module.exports.getUserAttendance = function(userid, currentClassYear){
     }
 
     function largeGroupFilter(a) {
-        return a.user.$oid === userid && a.classYear.$oid === currentClassYear._id.$oid && !a.smallgroup && !a.bonusDay;
+        return a.user.$oid === userid && a.classYear.$oid === currentClassYear._id.$oid && !a.smallgroup && !a.bonusDay && a.verified;
```
Comment (seveibar) [inline attendance.js:27]: brilliant!

**Comment by seveibar** [comment]
great job on this one aaron, really pulled through at the last second here!

**Comment by seveibar** [comment]
my only question, will it scale?

**Comment by seveibar** [comment]
also would you mind squashing these commits a bit?

**Comment by seveibar** [comment]
*commit

**Comment by agundy** [comment]
Verified to scale to millions of users. May have to optimize for billions of users

**Comment by seveibar** [comment]
If it can't scale to billions I'm not sure it's ready for deployment, maybe throw on dev for now?

**Comment by seveibar** [comment]
@kmcnellis could you weigh in?

**Comment by agundy** [comment]
@kmcnellis might know how to make verify it will scale

**Comment by seveibar** [comment]
I'm going to let @kmcnellis make the call here- let's have a conference call a bit later.

**Pull request merged**